src/collections/_documentation/workflow/notifications/alerts.md

```diff
@@ -3,22 +3,129 @@ title: 'Alerts'
 sidebar_order: 1
 ---
 
-Alerts are configured per project and are based on the rules defined for that project. To modify the rules visit your **Project Settings** > **Alerts** > **Rules** tab. On this page you’ll see a list of all active rules and can add alert rules or modify existing rules.
+Each time you create a new project, or a new issue occurs, Sentry creates a default alert rule. Alerts are generated based upon a project’s alert rules. By default, all project members receive alert notifications via email, and all integrations specified in [actions](#actions) receive alert notifications.
```

Comment:
"or a new issue occurs," << this part isn't true, the default rule is only generated upon project creation.